src/main/java/sparrow/data/task/Event.java

```diff
@@ -0,0 +1,31 @@
+package sparrow.data.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an Event in the task list.
+ */
+public class Event extends Task {
+    protected LocalDate date;
+
+    /**
+     * Creates an Event.
+     * @param description Details of the event.
```

Comment:
I think you can add a line in between the parameters and the description of the method